src/commonMain/kotlin/exchange.dydx.abacus/state/manager/Environment.kt

```diff
@@ -430,6 +430,7 @@ class V4Environment(
     val chainName: String?,
     val chainLogo: String?,
     rewardsHistoryStartDateMs: String,
+    val megavaultHistoryStartDateMs: Double?,
```

Comment:
yeah it's a string

Reply:
We will parse it as double.. since that's how it's going to be used.